api-spec/transactions-api.yaml

```diff
@@ -362,6 +362,10 @@ components:
           description: Cart identifier provided by creditor institution
           type: string
           example: idCartFromCreditorInstitution
+        orderId:
```

Comment:
I think that we can create an api `v2` version with mandatory field `orderId`.
This way we can add also other fields that will be required for NPG integration, such as `correlationId` and so on, making those fields mandatory.
What do you think? @infantesimone